src/bitmessagekivy/mpybit.py

```diff
@@ -206,6 +206,85 @@ def load_screen_callback(self, instance, dt=0):
             except Exception as e:
                 self.root.ids.id_trash.children[0].children[1].active = False
 
+    @staticmethod
+    def get_all_address_list():
+        """Getting list of all the addresses"""
+        addresses = [addr for addr in config.addresses()
+                     if config.getboolean(str(addr), 'enabled')]
+        return addresses
+
+    @staticmethod
+    def formatted_selected_address(selected_addr):
+        """Formatting selected address"""
+        address_list = NavigateApp.get_all_address_list()
```

Comment:
we only need this if `selected_addr` is `True` (i.e. defined), this way we save some CPU cycles